### PR TITLE
feat: make url prefix configurable

### PR DESCRIPTION
**Pull request by fengelniederhammer**
This is necessary when deploying on a server with a proxy in between that remaps paths (e.g. example.server/lapis/aggregated -> 127.0.0.1/aggregated)

### CODE REVIEW

File: lapis2/docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     image: ghcr.io/genspectrum/lapis-v2:${LAPIS_TAG}
     ports:
       - "8080:8080"
-    command: --silo.url=http://silo:8081 --lapis.databaseConfig.path=databaseConfig.yaml
+    command: --silo.url=http://silo:8081 --lapis.database-config.path=databaseConfig.yaml
```
Comment (JonasKellerer) [inline lapis2/docker-compose.yml:7]: What is the reason behind this naming change?

**Review by JonasKellerer — APPROVED** [review]

File: lapis2/docker-compose.yml (hunk 1 above)
Comment (JonasKellerer) [reply lapis2/docker-compose.yml:7]: Seems to be kebab-case for application properties.

**Review by TobiasKampmann — APPROVED** [review]
As far as I understand it, seems fine

**Comment by fengelniederhammer** [comment]
Closing in favor of #343

**Pull request closed without merging**